things/ugbasic.pldb

```diff
@@ -0,0 +1,18 @@
+title ugBASIC
+appeared 2021
+type pl
+isDead false
+photo https://retroprogramming.iwashere.eu/_media/ugbasic:diagrammi2-general.png
+demoVideo https://www.youtube.com/watch?v=Klg1njCwZhs
+creators Marco Spedaletti
+description ugBASIC is an isomorphic and open source language, fully documented and designed to develop portable programs, without sacrificing efficiency. With a single source it is therefore possible to create games for numerous 8 bit platforms.
+website https://ugbasic.iwashere.eu
+documentation https://ugbasic.iwashere.eu/#about
+fileExtensions .bas
+country Italia
```

Comment:
You might want to change this to Italy. Italia is the name of the country in Italian.

Reply:
Thanks :)